GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -30,8 +30,8 @@ public partial class RevisionDiffControl : GitModuleControl
             new("Are you sure you want to delete the selected file(s)?");
         private readonly TranslationString _deleteFailed = new("Delete file failed");
         private readonly TranslationString _multipleDescription = new("<multiple>");
-        private readonly TranslationString _selectedRevision = new("Second: b/");
-        private readonly TranslationString _firstRevision = new("First: a/");
+        private readonly TranslationString _selectedRevision = new("Second(B): ");
+        private readonly TranslationString _firstRevision = new("First(A): ");
```

Comment:
```suggestion
        private readonly TranslationString _selectedRevision = new("Second: B ");
        private readonly TranslationString _firstRevision = new("First: A ");
```
This matches the FileStatusList headings better.

Reply:
OK